eqcorrscan/utils/clustering.py

```diff
@@ -79,32 +89,46 @@ def cross_chan_correlation(st1, streams, shift_len=0.0, xcorr_func='fftw',
         # We need to copy it first.
         streams = [stream.copy() for stream in streams]
     # Check which channels are in st1 and match those in the stream_list
-    st1, streams, stream_indexes = _prep_data_for_correlation(
+    st1, prep_streams, stream_indexes = _prep_data_for_correlation(
```

Comment:
I like this - renaming to `prep_streams` - if you want you could also rename `st1` to something else to make it clear that `st1` and the output of this are different.